django/contrib/admin/templatetags/base.py

```diff
@@ -4,19 +4,20 @@
 
 
 class InclusionAdminNode(InclusionNode):
+    """
+    Admin template tag that allows its template to be overridden per model,
+    per app, or globally.
+    """
+
     def __init__(self, parser, token, func, template_name, takes_context=True):
         self.template_name = template_name
-
         params, varargs, varkw, defaults, kwonly, kwonly_defaults, _ = getfullargspec(func)
-        if len(params) > 0 and params[0] == 'self':
```

Comment:
I'm not sure the purpose of this but tests seem to pass without it.

Reply:
Oh, it's a PY2 relics...